pkgs/development/python-modules/graspy/default.nix

```
@@ -34,12 +36,11 @@ buildPythonPackage rec {
     seaborn
   ];
 
-  checkInputs = [ pytest pytestcov ];
+  checkInputs = [ pytestCheckHook pytestcov ];
 
   checkPhase = ''
     runHook preCheck
```

since you used pytestCheckHook, you can replace the current checkPhase
```suggestion
  pytestFlagsArray = [ "tests" "--ignore=docs" ];
  disabledTests = [ "gridplot_outputs" ];
```

---

Fixed, thanks!